python/plugins/processing/ui/resultsdockbase.ui

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>DockWidget</class>
+ <widget class="QDockWidget" name="DockWidget">
```

Comment:
> Of course I did. Here is hoiw the .ui file looks

sorry - didn't mean to state the obvious :p 

I'm not sure then... I can't think what's happening here